emsdk.py

```diff
@@ -529,6 +529,8 @@ def untargz(source_filename, dest_dir):
 def fix_potentially_long_windows_pathname(pathname):
   if not WINDOWS:
     return pathname
+  if MSYS:
```

Comment:
Combine this with the above block?  `if not WINDOWS or MSYS:`?

Reply:
Do you know what happens if the path length is above 260 chars?  This prefix seems to specifically be about issues with  such long filenames.

Reply:
It fails either way, even if I change line 539 from `if MINGW:` to `if MINGW or MSYS:` so that it no longer fails with "cannot find the file specified". It then still fails with "The filename or extension is too long: '//?/D:/Development/emsdkdeep/abcdefghijklmnopqrstuvxyz/abcdefghijklmnopqrstuvxyz/abcdefghijklmnopqrstuvxyz/abcdefghijklmnopqrstuvxyz/abcdefghijklmnopqrstuvxyz/abcdefghijklmnopqrstuvxyz/abcdefghijklmnopqrstuvxyz/emsdk/node/unzip_temp/node-v14.18.2-win-x64'"

So the fix for windows does not seem to work for the msys environment

Reply:
There is some hidden layer that messes with paths but does not understand the prefix that is intended to circumvent MAX_PATH. There is also documentation on msys2 on path related conversions: https://www.msys2.org/docs/filesystem-paths/#windows-unix-path-conversion

But I failed to devices a fix either by configuration or by code that is able to handle the prefix / long paths.

Reply:
Perhaps on MSYS we should just error out here bigger than MAX_PATH?  Since `cannot find the file specified` is less useful of an error message.

Happy to land this change either way.